skills/custom-devotional-crafter/scripts/push_custom_devotional.py

```
@@ -0,0 +1,69 @@
+
+    req = urllib.request.Request(
+        endpoint,
+        data=body,
+        method="POST",
+        headers={
+            "Content-Type": "application/json",
+            "apikey": service_role,
+            "Authorization": f"Bearer {service_role}",
+            "Prefer": "resolution=merge-duplicates,return=representation",
+        },
+    )
```

Ruff (0.15.11)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)